lib/modules/platform/pr-body.ts

```diff
@@ -9,6 +10,14 @@ export const prDebugDataRe = regEx(
   /\n?<!--renovate-debug:(?<payload>.*?)-->\n?/
 );
 
+export const renovateConfigHashRe = regEx(
```

Comment:
same here, why export?

Reply:
I was originally intended to use these in tests.
will be removed